AitZak/alyz
# [Evolution][Back] Implemented ApiPlatform, and implemented User Authe…

Requirements :   
composer update
php bin/console d:s:u --f

Description : 
Implemented ApiPlatform with basic Settings(look at : localhost:80/api for a swagger view).
Configured Symfony Security Bundle.
Implemented User Authentication with JWT.
How to test ? 
Postman : 
REGISTRATION : 
method : POST          http://localhost:80/api/users
                                "email" : 'your_email',
                                 "lastname": 'your_lastname',
                                  "firstname": 'your_firstname',
                                  "password": 'your_password'
LOGIN : POST          http://localhost:80/api/login_check
                                "username" : 'your_email',
                                  "password": 'your_password'
SHOULD RETURN A TOKEN!!!
Paste this token in postman Authorization section with Bearer Token Type.
And try to visit another route (for example : /api/sings or get /api/users/.
